Header header
float32 x
float32 gamma 